From af63656b7fd67591e9e60867a59003979e67b031 Mon Sep 17 00:00:00 2001
From: Nadav Haklai <nadavh@marvell.com>
Date: Mon, 28 Jan 2019 22:20:17 +0200
Subject: [PATCH 0918/1051] arm64: change MMIO_64BIT default config option to y

Align this config option with standard arm64 behaviour.
Change to n only for platforms cannot support 64bit register
access mode.

Change-Id: I693d79f9a3aa2f5424dcbf9a504c113be2dad232
Signed-off-by: Nadav Haklai <nadavh@marvell.com>
Reviewed-on: https://sj1git1.cavium.com/2870
Tested-by: sa_ip-sw-jenkins
Reviewed-by: Andrew Pinski <apinski@cavium.com>
Reviewed-by: Nadav Haklai <Nadav.Haklai@cavium.com>
[Kevin: The original patch got from Marvell sdk10.0_19.06]
Signed-off-by: Kevin Hao <kexin.hao@windriver.com>
---
 arch/arm64/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/Kconfig b/arch/arm64/Kconfig
index 752ebfea963d..03c910a2154e 100644
--- a/arch/arm64/Kconfig
+++ b/arch/arm64/Kconfig
@@ -703,7 +703,7 @@ config CPU_BIG_ENDIAN
 
 config MMIO_64BIT
 	bool "64BIT MMIO Access"
-	default n
+	default y
 	help
 	  Say Y if you your platform supports 64bit MMIO access.
 
-- 
2.17.1

